ihatemoney/conf-templates/apache-vhost.conf.j2

```diff
@@ -4,13 +4,15 @@
 
     WSGIDaemonProcess ihatemoney user=www-data group=www-data threads=5 python-path={{ pkg_path }} {% if venv_path %}python-home={{ venv_path }}{% endif %}
     WSGIScriptAlias / {{ pkg_path }}/wsgi.py
+    WSGIPassAuthorization On
 
     ErrorLog /var/log/apache2/ihatemoney.example.com_error.log
     CustomLog /var/log/apache2/ihatemoney.example.com_access.log combined
 
     <Directory {{ pkg_path }}>
        WSGIProcessGroup ihatemoney
        WSGIApplicationGroup %{GLOBAL}
+       AuthType Basic
```

Comment:
With this line, no risk that it prompts user for auth even on non-API page ? (just asking).